src/renderer/components/RecentConnections.tsx

```diff
@@ -32,54 +55,42 @@ const RecentConnections = () => {
       <div className="recent-wrapper">
         <h1>Recent Connections</h1>
         <div className="d-flex justify-content-center">
-          <Table className="table">
-            <thead>
-              <tr>
-                <th>Nick Name</th>
-                <th>Database Type</th>
-                <th>Address</th>
-                <th>Port</th>
-                <th>User Name</th>
-                <th>Actions</th>
-              </tr>
-            </thead>
-            <tbody>
-              {connect.map((value) => {
-                if (value.connectionConfig.config === 'manual') {
+          <form onSubmit={submitForm}>
+            <Table className="table">
+              <thead>
+                <tr>
+                  <th>Nick Name</th>
+                  <th>Database Type</th>
+                  <th>Address</th>
+                  <th>Port</th>
+                  <th>User Name</th>
+                  <th>Actions</th>
+                </tr>
+              </thead>
+              <tbody>
+                {connect.map((value) => {
                   return (
-                    <tr key={value.id}>
-                      <td>
-                        <Link to="/Execute">
-                          <button
-                            className="buttonSelect"
-                            type="button"
-                            onClick={() => handleSelect(value.id)}
-                          >
-                            {value.nickname}
-                          </button>
-                        </Link>
-                      </td>
-                      <td>{value.type}</td>
-                      <td>{value.connectionConfig.address}</td>
-                      <td>{value.connectionConfig.port}</td>
-                      <td>{value.connectionConfig.username}</td>
-                      <td>
-                        <EditForm config={value} setConnect={getConnections} />
-                        <button
-                          type="button"
-                          className="deleteButton"
-                          onClick={() => handledelete(value.id)}
-                        >
-                          <Trash />
-                        </button>
-                      </td>
-                    </tr>
+                    <>
+                      {edit === value.id ? (
+                        <EditRow
+                          value={value}
+                          toggleReadOnly={handleCancel}
+                          handleChange={submitForm}
+                        />
+                      ) : (
+                        <ReadRow
+                          value={value}
+                          handleDelete={handledelete}
```

Comment:
```suggestion
                          handleDelete={handleDelete}
```
Change the function definition name as well